R/render_report.R

```diff
@@ -15,13 +15,44 @@
 #'
 #' @return
 #' @export
-render_report <- function(output = NULL, prams = "ask") {
+render_summary_report <- function(output = NULL, params = "ask") {
 
   if (is.null(output)) {
     rlang::abort("please provide an output file path")
   }
 
-  report_pth <- system.file("analysis_report.Rmd", package = "d.inspectEHR")
-  rmarkdown::render(report_pth, params = prams,
-                    output_file = output)
+  report_pth <- system.file("OMOP-Summary.Rmd", package = "ccaa.inspectEHR")
+  rmarkdown::render(report_pth, params = params,
+                    output_file = output,
+                    envir = new.env())
+}
+
+
+#' Render Quality Report
+#'
+#' @param output an output file path
+#' @param prams a string "ask", which will launch an interactive window to enter
+#'   database connection details, or a list that contains these details:
+#'
+#'   - driver: string, can be any of: "PostgreSQL", "ODBC", or "SQLite"
+#'   - host: string, host name
+#'   - port: integer, port number
+#'   - dbname: string, database name
+#'   - schema: string, database target schema
+#'   - user: string, database username
+#'   - password: string, database user password
+#'   - local_hospital: string, local hospital
+#'
+#' @return
+#' @export
+render_quality_report <- function(output = NULL, params = "ask") {
```

Comment:
Can't you ask for the output path to be part of the params?

Reply:
That will reduce the number of arguments needed.

Reply:
This is done!